.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+ruby-2.4.1
```

Comment:
I think you need to remove `ruby`; this should just be the version numbers, yes?